MotionMark1.2/tests/master/resources/image-data.js

```diff
@@ -170,7 +170,7 @@ var ImageDataStage = Utilities.createSubclass(Stage,
                 context.putImageData(imageData, 0, 0);
             else {
                 this._refreshElement(element);
-                element.getContext("2d").drawImage(Stage.randomElementInArray(this.images), 0, 0, this.imageWidth, this.imageHeight);
+                element.getContext("2d", { willReadFrequently: true }).drawImage(Stage.randomElementInArray(this.images), 0, 0, this.imageWidth, this.imageHeight);
```

Comment:
Suggestion: Changing this line is not necessary because the `options` argument only has an effect on the first call to `getContext`. IMHO, it would be cleaner if we just used `context` instead of `element.getContext(...)` here.

Reply:
Done. I left it as `element.getContext` but can switch to `context` in a follow up.